packages/app/src/sandbox/index.js

```diff
@@ -65,6 +68,15 @@ requirePolyfills().then(() => {
       // Means we're in the editor
       setupHistoryListeners();
       setupConsole();
+      window.addEventListener('message', ({ data }) => {
+        switch (data.type) {
+          case 'activate':
```

Comment:
I just picked "activate" off the top of my head by the way. I'm not very familiar with the CS client so I don't know if it's guaranteed to be unique or what else might be posting messages to this `iframe`.

Reply:
Yap! This is fine! We can otherwise do a check on `data.codesandbox == null`.